packages/unmock-core/src/service-loaders/index.ts

```diff
@@ -0,0 +1,24 @@
+import debug from "debug";
+import * as _ from "lodash";
+import { FsServiceDefLoader } from "../fs-service-def-loader";
```

Comment:
If we have `/service-loaders/`, maybe we can move `fs-service-def-loader` to this folder as well...

Reply:
👍